packages/ui/__stories__/NumberBadge.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { NumberBadge } from '../src/NumberBadge'
+
+export default {
+  title: 'components/NumberBadge',
+  component: NumberBadge,
+}
+
+export const Default = () => <NumberBadge number={1} />
+
+export const LessThan99 = () => <NumberBadge number={99} />
```

Comment:
nit: 99 is not lesser than 99 😄 
```suggestion
export const LessThan100 = () => <NumberBadge number={99} />
```